perf/sawtooth/sawtooth_workload/src/main.rs

```diff
@@ -0,0 +1,112 @@
+extern crate clap;
+extern crate sawtooth_sdk;
+
+mod batch_gen;
+
+use std::fs::File;
+use std::io;
+use std::io::prelude::*;
+use std::error::Error;
+
+use batch_gen::generate_signed_batches;
+use clap::{App, ArgMatches, AppSettings, Arg, SubCommand};
+
+fn main() {
+    let arg_matches = 
+        App::new("sawtooth-perf")
```

Comment:
Maybe use the cargo info here:

const APP_NAME: &'static str = env!("CARGO_PKG_NAME");
const VERSION: &'static str = env!("CARGO_PKG_VERSION");